### DISCUSSION TITLE
sig patch Wii U error wup installer

### DISCUSSION THREAD

**Discussion by Thomassss251** [community/community#88398] [Programming Help]
### Select Topic Area

Question

### Body

Bonjour, j'ai un gros probleme, j'ai un soucis de sig patch avec la wiiu pour installer certains jeux, auriez vous une solution je debute dans le domaine du hack et je ne veux pas perdre de temps ....??

Hello, I have a big problem, I have a sig patch problem with the wiiu to install some games, do you have a solution I am new to the hack field and I don't want to waste time....??

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Thomassss251 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by CGdiego** [comment]
Hello, you can get Sigpatches Module here: https://github.com/marco-calautti/SigpatchesModuleWiiU/releases.
